templates/schema.sql

```diff
@@ -0,0 +1,45 @@
+CREATE DATABASE auctionshop
```

Comment:
Можно сделать так, чтобы каждый раз не ловить ошибку.
```suggestion
CREATE DATABASE IF NOT EXISTS auctionshop
```